creator-node/test/fileManager.js

```diff
@@ -149,4 +163,113 @@ describe('test saveFileToIpfsFromFs', () => {
 
     assert.deepStrictEqual(originalSegmentBuf.compare(ipfsResp), 0)
   })
+
+  // ~~~~~~~~~~~~~~~~~~~~~~~~~ saveFileFromBuffer() TESTS ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  /**
+   * Given: a file buffer is being saved to ipfs, fs, and db
+   * When: cnodeUserUUID is not present
+   * Then: an error is thrown
+   */
+  it('should throw error if cnodeUserUUID is not present', async () => {
```

Comment:
the name of the function should be in the test string here. otherwise unless you open the code and look for comments it wont be clear what will error if cnodeUserUUID isn't present. like in circleci for instance

Reply:
you could also have a new nested describe block